lib/ja_serializer/dsl.ex

```diff
@@ -0,0 +1,287 @@
+defmodule JaSerializer.DSL do
+  alias JaSerializer.Relationship.{HasMany,HasOne}
```

Comment:
This is cool, what is this tuple like syntax and what version support does it have with elixir?